server/tests/trip.test.js

```diff
@@ -146,7 +147,7 @@ describe('Trip test suite', () => {
         done(err);
       });
   });
-  it('8.should not get a single trip which doesnt beloge to you', (done) => {
+  it('8.should not get a single trip which does not belong to you', (done) => {
     router()
       .get('/api/v1/trips/1')
       .set('token', notMangerToken)
```

Comment:
Update the variable name here from notMangerToken to notManagerToken instead. That's a grammatical error.

Reply:
updated